contracts/diamond/presets/OwnableDiamond/OwnableDiamondStorage.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ *  @notice The storage contract of Ownable Diamond preset
+ */
+contract OwnableDiamondStorage {
+    bytes32 public constant OWNABLE_DIAMOND_STORAGE_SLOT =
+        keccak256("diamond.standard.ownablediamond.storage");
+
+    struct ODStorage {
+        address owner;
+    }
+
+    function getOwnableDiamondStorage() internal pure returns (ODStorage storage ods) {
+        bytes32 slot = OWNABLE_DIAMOND_STORAGE_SLOT;
```

Comment:
What about our codestyle agreements?)

Reply:
Haven't added new codestyle to the modules yet